src/Admin/SupportScreen.php

```diff
@@ -182,16 +182,44 @@ public function enqueue_assets( $hook_suffix ) {
 				'var ampSupport = %s;',
 				wp_json_encode(
 					[
-						'restEndpoint' => get_rest_url( null, 'amp/v1/send-diagnostic' ),
-						'args'         => $args,
-						'data'         => $data,
+						'restEndpoint'          => get_rest_url( null, 'amp/v1/send-diagnostic' ),
+						'args'                  => $args,
+						'data'                  => $data,
+						'ampValidatedPostCount' => $this->get_amp_validated_post_counts(),
 					]
 				)
 			),
 			'before'
 		);
 	}
 
+	/**
+	 * Get count of amp validated post.
+	 *
+	 * @return array [
+	 *     @type int $all Count of all AMP validated URL post.
+	 *     @type int $valid Count of all AMP validated URL post that are not stalled.
+	 * ]
+	 */
+	public function get_amp_validated_post_counts() {
+
+		$amp_validated_post_count = wp_count_posts( AMP_Validated_URL_Post_Type::POST_TYPE_SLUG );
+
+		$query_args = [
+			'post_type'      => AMP_Validated_URL_Post_Type::POST_TYPE_SLUG,
+			'posts_per_page' => 1,
+			'post_status'    => 'publish',
+			'meta_key'       => AMP_Validated_URL_Post_Type::VALIDATED_ENVIRONMENT_POST_META_KEY,
+			'meta_value'     => maybe_serialize( AMP_Validated_URL_Post_Type::get_validated_environment() ), // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_value
+		];
+		$query      = new WP_Query( $query_args );
+
+		return [
+			'all'   => intval( $amp_validated_post_count->publish ),
+			'valid' => intval( $query->found_posts ),
```

Comment:
Would `fresh` be a more accurate name for this? Or conversely, how about inverting the logic on the frontend and returning here `stale` count instead of `valid`/`fresh`?